assets/js/util/tracking/index.js

```diff
@@ -30,13 +30,23 @@ const {
 	userIDHash,
 } = global._googlesitekitBaseData || {};
 
+const {
+	user: { roles: currentUserRoles },
+	authenticated: authenticatedUser,
+} = global._googlesitekitUserData || {};
+
+const { GOOGLESITEKIT_VERSION: pluginVersion } = global;
+
 const initialConfig = {
 	activeModules,
 	trackingEnabled,
 	trackingID,
 	referenceSiteURL,
 	userIDHash,
 	isSiteKitScreen,
+	currentUserRoles,
+	authenticatedUser,
```

Comment:
Let's use `isAuthenticated` for consistency with other uses/references in JS.
```suggestion
	isAuthenticated,
```

Reply:
👍